### ISSUE TITLE
deleting collections is not working

### ISSUE THREAD

**Issue by falmanna** [bcc-code/directus-schema-sync#47]
### Describe the Bug

deleting collections is not working when schema split is enabled

### To Reproduce

sync deleting a collection with schema split is enabled

### Version

2.1.2

### Installed Extension Via

NPM Custom

**Comment by u12206050** [comment]
You might need to manually delete the file of the deleted collection since there is nothing that tracks it.

**Comment by u12206050** [comment]
Could it be related to this: https://github.com/directus/directus/issues/16683 ?

**Comment by falmanna** [comment]
@u12206050 yes I had this issue and it was happening even when using directus snapshot cli. 
But the issue here is about top level collections are not getting deleted when splitting is enabled.

**Comment by falmanna** [comment]
Yeah haven't had this in a while
